src/client/activity/UserActivitySearch.js

```diff
@@ -122,12 +124,26 @@ const rewardsFilters = [
   filterValues[accountHistoryConstants.CLAIM_REWARDS],
 ];
 
-@connect(null, {
-  updateAccountHistoryFilter,
-})
+@withRouter
+@connect(
+  (state, ownProps) => ({
+    user: ownProps.isCurrentUser
+      ? getAuthenticatedUser(state)
+      : getUser(state, ownProps.match.params.name),
+  }),
+  {
+    updateAccountHistoryFilter,
+  },
+)
 class UserActivitySearch extends React.Component {
   static propTypes = {
+    isCurrentUser: PropTypes.bool, // eslint-disable-line react/no-unused-prop-types
```

Comment:
And there

Reply:
Used in line 130 on this file, eslint didn't recognize it :(

Reply:
Used in line 32 and 41 above